server/routes/routerRequestHandlers/retrieveCuriousInPrisonCourses.ts

```diff
@@ -10,18 +9,9 @@ const retrieveCuriousInPrisonCourses = (curiousService: CuriousService): Request
   return asyncMiddleware(async (req: Request, res: Response, next: NextFunction) => {
     const { prisonNumber } = req.params
 
-    // Lookup the prisoners In Prison Courses and store in res.locals if its either not there, or is for a different prisoner
```

Comment:
This relates to my answer to the comment above - this block of logic was completely pointless because it is using `res.locals` rather than the session!